wire-ios/Wire-iOS/Sources/UserInterface/Calling/Haptic Feedback/CallHapticsController.swift

```diff
@@ -83,7 +83,7 @@ final class CallHapticsController {
 
     private func updateParticipantsVideoStateList(_ newParticipants: [CallParticipant]) {
         let newVideoStates = createVideoStateMap(using: newParticipants)
-        Log.haptics.debug("updating video state map: \(newVideoStates), old: \(videoStates)")
+        Log.haptics.debug("updating video state map")
```

Comment:
Those video changes are very large objects in the log with hundreds of lines of the core data object array. To make the debug life easier, I have removed the states from the log.